specs/eventing/control-plane.md

```diff
@@ -134,6 +134,10 @@ an associated Trigger destination which does not currently have a `true`
 `Ready` condition, including events received by the Broker before the Trigger
 was created.
 
+If the Broker's `spec.delivery.deadLetterSink` field it set, it MUST be
+resolved to a URI and reported in `status.deadLetterSinkUri` in the same manner
+as the `status.address.url` field before setting the `Ready` condition to `true`.
```

Comment:
```suggestion
resolved to a URI and reported in `status.deadLetterSinkUri` before setting the `Ready` condition to `true`.
```

Reply:
The reference to `status.address.url` didn't seem to help anyone understand the pattern.